src/tests/ftest/erasurecode/online_rebuild.py

```diff
@@ -32,6 +32,7 @@ def test_ec_online_rebuild(self):
         :avocado: tags=hw,large
         :avocado: tags=ec,ec_array,ec_online_rebuild,rebuild
         :avocado: tags=test_ec_online_rebuild_array
+        :avocado: tags=test_ec_online_rebuild
```

Comment:
Can we also remove the incorrect `test_ec_online_rebuild_array` above?

Reply:
Done.